### PR TITLE
A mix of smaller adjustments

### PR DESCRIPTION
**Pull request by kristofferlarberg**
- Style `NotFound` with styled components
- Create indicator of script textlength
- Set height for `ScriptBox` when placeholder text
- Lower sharpness of images
- Edit metadata

### CODE REVIEW

File: src/components/Script.jsx [hunk 1]
```diff
@@ -41,33 +45,86 @@ const ScriptBox = styled.section`
...
+  mobile, open, position, text, textLength,
 }) => {
   const [openScript, setOpenScript] = useState(text === 'Nothing here...');
+  const [lengthyText, setLengthyText] = useState(false);
+
+  useEffect(() => {
+    if (textLength > 1000) {
+      setLengthyText(true);
+    }
+  }, [textLength, lengthyText]);
```
Comment (calmfors) [inline src/components/Script.jsx:71]: Are useEffect and useState on lengthyText really necessary? The value does not change during component lifecycle.

File: src/components/Script.jsx [hunk 2]
```diff
@@ -41,33 +45,86 @@ const ScriptBox = styled.section`
...
+  color: ${props => (!props.lengthyText ? 'var(--lightgrey)' : 'red')};
+  margin: 0;
+  width: auto;
+`;
+
 const Script = ({
-  mobile, open, position, text,
+  mobile, open, position, text, textLength,
 }) => {
   const [openScript, setOpenScript] = useState(text === 'Nothing here...');
```
Comment (calmfors) [inline src/components/Script.jsx:63]: Should be the same text as below ('No content here at the moment.')

File: src/components/Script.jsx [hunk 3]
```diff
@@ -41,33 +45,86 @@ const ScriptBox = styled.section`
...
         tabIndex={0}
+        textLength={textLength}
       >
-        {text}
+        {textLength ? (
+          <>
+            <RichText
+              linkResolver={linkResolver}
+              render={text}
+            />
```
Comment (calmfors) [inline src/components/Script.jsx:90]: No underline on the text?

File: src/components/Script.jsx [hunk 4]
```diff
@@ -41,33 +45,86 @@ const ScriptBox = styled.section`
...
+            <AmountContainer>
+              <Amount lengthyText={lengthyText}>
+                {`${textLength} characters`}
+              </Amount>
+            </AmountContainer>
```
Comment (calmfors) [inline src/components/Script.jsx:121]: Should it say text length even when it's ok (below 1000)? And wasn't the text supposed to be cropped to 1000 characters anyway?

File: src/pages/Work.jsx [hunk 5]
```diff
@@ -321,14 +321,8 @@ const Work = ({match}) => {
...
-            text={workData.work_script.length ? (
-              <RichText
-                linkResolver={linkResolver}
-                render={workData.work_script}
-              />
-            ) : (
-              'Nothing here...'
-            )}
+            text={work.work_script}
+            textLength={RichText.asText(work.work_script).length}
```
Comment (calmfors) [inline src/pages/Work.jsx:325]: Should be workData.work_script

**Review by calmfors — COMMENTED** [review]
Why is the text not cropped to max 1000 characters? Mid sentence or keep last sentence even if text exceeds 1000.

File: src/pages/Work.jsx (hunk 5 above)
Comment (kristofferlarberg) [reply src/pages/Work.jsx:325]: This is fixed in later commit.

File: src/components/Script.jsx (hunk 4 above)
Comment (kristofferlarberg) [reply src/components/Script.jsx:121]: Atm grey when OK and red when NOT OK. But if we shorten the text then maybe only the red indicator (or nothing)..

File: src/components/Script.jsx (hunk 3 above)
Comment (kristofferlarberg) [reply src/components/Script.jsx:90]: Felt like a too big of an intrusion in the text…

File: src/components/Script.jsx (hunk 2 above)
Comment (kristofferlarberg) [reply src/components/Script.jsx:63]: Ah ofc!

File: src/components/Script.jsx (hunk 1 above)
Comment (kristofferlarberg) [reply src/components/Script.jsx:71]: No you're right, I'll replace it with:
`
const lengthyText = textLength > 1000 && true;`

File: src/components/Script.jsx (hunk 2 above)
Comment (kristofferlarberg) [reply src/components/Script.jsx:63]: Edit: Not sure how to think with this. Let's talk more about this aside from the text length issue.

**Pull request merged**